### PR TITLE
capitalize sql keywords

### PR DESCRIPTION
**Pull request by PingCAPer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Capitalized 2 SQL keywords.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
Could you please also capitalize the SQL keywords in line 188? Thanks~

**Comment by PingCAPer** [comment]
> Could you please also capitalize the SQL keywords in line 188? Thanks~

Sure.

File: metrics-schema.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ The following statement queries the P99 time within the range of [`2020-03-25 23
 {{< copyable "sql" >}}
 
 ```sql
-select * from metrics_schema.tidb_query_duration where value is not null and time>='2020-03-25 23:40:00' and time <= '2020-03-25 23:42:00' and quantile=0.99;
+SELECT * FROM metrics_schema.tidb_query_duration where value is not null and time>='2020-03-25 23:40:00' and time <= '2020-03-25 23:42:00' and quantile=0.99;
```
Comment (CharLotteiu) [inline metrics-schema.md:159]:
```suggestion
SELECT * FROM metrics_schema.tidb_query_duration WHERE value is not null AND time>='2020-03-25 23:40:00' AND time <= '2020-03-25 23:42:00' AND quantile=0.99;
```

**Comment by PingCAPer** [comment]
@CharLotteiu PTAL again, thanks!

**Review by CharLotteiu — APPROVED** [review]

**Comment by CharLotteiu** [comment]
LGTM.

**Pull request closed without merging**